tasks/install-on-Darwin.yml

```diff
@@ -13,13 +13,24 @@
     state: latest
     update_homebrew: yes
 
+- name: Configure Buildkite service
+  template:
+    src: "homebrew.mxcl.buildkite-agent.plist.j2"
+    dest: "{{ buildkite_agent_conf_dir[ansible_os_family] }}/homebrew.mxcl.buildkite-agent.plist"
```

Comment:
Is it launchd valid to place a launchd plist outside of one of

```
~/Library/LaunchAgents
/Library/LaunchAgents
/Library/LaunchDaemons
/System/Library/LaunchAgents
/System/Library/LaunchDaemons
```

?

Reply:
When passing a plist to launchd with `brew services (re)?start service <plist>`, the plist is automatically copied to `~/Library/LaunchAgents` by brew.